### DISCUSSION TITLE
[Section 3.16 - Terraform import]     Referente a import de serviços na AWS

### DISCUSSION THREAD

**Discussion by Alves0611** [chgasparoto/curso-aws-com-terraform#134] [Q&A]
Olá, gostaria de saber se é possível migrar serviços(com conteudo) de uma conta AWS para outra utilizando o Terraform?

**Comment by chgasparoto** [comment]
@Alves0611, o que seria esse conteúdo?

**Reply by Alves0611** [reply]
@chgasparoto   um RDS com dados, um s3 com arquivos dentro ...

**Reply by chgasparoto** [reply]
Se os arquivos do S3 foram criados pelo Terraform, sim. Caso contrário, não. 
Para o RDS você teria que fazer a migração dos dados.
